qemu/tests/migration_with_block.py

```diff
@@ -261,7 +261,8 @@ def ping_pong_migration(repeat_times):
                 migration_thread.start()
                 if not utils_misc.wait_for(lambda: (
                         bool(vm.monitor.query("migrate")) and (
-                        'completed' != vm.monitor.query("migrate")['status'])), 30):
+                        'completed' != vm.monitor.query("migrate")['status'])),
+                                           60, 30):
```

Comment:
Modify to `60` is okay, but I think the first check should not be `30`.

Reply:
I think you should define the `step` parameter, instead of `first`.

Reply:
the point is, need to wait for few seconds to start, or the "status" info could not find with the output of migrate command, I tried and cut it from 30 to 10